### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewProblems_ColorContrast: Color Contrast ratio is less than 4.5:1 for placeholder for Absys theme. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52602]
Environment Details:
VSCode Version : 1.24.1
OS Version : Win10

Additional Details:
MAS Violated: MAS1.4.3,ColorContrast

Repro Steps:
1)Launch VS Code.
2) Click on View menu item and navigate to Problems
3)Navigate to filter text box.
4)Open Color Contrast anlyzer and check the ratio for placeholder.

Actual:
Color contrast ratio is less than 4.5:1 for the placeholder.
Same issue has been observed in below themes as well:
Light
Light+
Quiet Light
Dark+
Kimbie Dark
Monokai
Solarized Dark
Tomorrow Night Blue

Expected:
Color Contrast ratio should be equal to or greater than 4.5:1

Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

User Impact:
The user will not be able to read/see the placeholder.

MAS Reference:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)

Attachment for reference: 
![accessibility a11y_vscode_viewproblems_colorcontrast](https://user-images.githubusercontent.com/39204909/41718497-c4324a12-757a-11e8-8684-a2da8acb4562.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52593
